### PR TITLE
Improve fixed scroll with animation

### PR DESCRIPTION
**Pull request by sergomen**
https://github.com/lcrojano/Giphy_Explorer/assets/18175528/d1b818a2-da1a-43f6-95c2-34fbfe6db3af

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -76,15 +75,36 @@ <h1 class="grow-2">Giphy Search</h1>
     <aside></aside>
     <script type="text/javascript">
         mybutton = document.getElementById("myBtn");
+        logo = document.getElementById("logo");
```
Comment (lcrojano) [inline index.html:84]: Managing features through JavaScript is a great idea.

**Review by lcrojano — APPROVED** [review]

File: styles/styles.css [hunk 2]
```diff
@@ -92,265 +92,223 @@ h1 {
...
-  justify-content: center;
-  align-items: center;
-  width: 20px;
-  height: 20px;
-  vertical-align: middle;
-  background-position: center;
-}
-#github-link h1 {
-  font-size: 10px;
-  font-family: sans-serif;
```
Comment (lcrojano) [inline styles/styles.css:332]: The deletion of this particular lines will result in breaking the GitHub link feature which was introduced in issue #38.

**Review by lcrojano — CHANGES_REQUESTED** [review]
Congratulations! Your PR successfully implements the requested feature. However, there is a conflict that needs to be avoided. Please check the lines that implement styles for the github link.

**Comment by sergomen** [comment]
Do I need to do something with these css lines?

**Pull request merged**